src/Microsoft.Framework.DependencyInjection/ServiceProvider.cs

```diff
@@ -143,16 +144,28 @@ internal IServiceCallSite GetResolveCallSite(IService service, ISet<Type> callSi
             }
         }
 
+        internal void Reset()
+        {
+            if (_offlineDisposables == null)
+            {
+                throw new InvalidOperationException("ServiceProvider reused before being disposed");
```

Comment:
Do we have a resx in this project? Programmatic names like `ServiceProvider` need to be formatted in with parameters.